react-app/src/components/ProposalDetailScreen/ProposalDetailScreenAPI.ts

```diff
@@ -75,7 +75,10 @@ export function useProposalQuery(): {
             proposal.votingStartTime &&
             proposal.votingEndTime &&
             differenceInDays(new Date(proposal.votingEndTime), Date.now()),
-          depositTotal: convertMinimalTokenToToken(proposal.depositTotal),
+          depositTotal: proposal.depositTotal.map((t) => ({
+            denom: t.denom,
```

Comment:
`t.denom` would be a minimal token denom, but since amount is converted from minimal token to token, token denom should be used here?

Reply:
oh that's right